Firestore/core/src/firebase/firestore/nanopb/nanopb_string.h

```diff
@@ -73,7 +74,7 @@ class String : public util::Comparable<String> {
   }
 
   ~String() {
-    delete bytes_;
+    std::free(bytes_);
```

Comment:
Nice catch. Do you think it would be worth moving this destructor into the .cc to avoid bleeding `#include <cstdlib>` into all consumers?

Reply:
Maybe? Realistically, stdlib.h is pretty much guaranteed to be pulled in regardless. But another decent reason to move it might be to keep the malloc/free in the same file (so it's more obvious when a mismatch occurs.) Moved.

Reply:
That's a much better reason than the one I offered :-).